_announcements/01_apple_common_removal.md

```diff
@@ -0,0 +1,319 @@
+---
+layout: default
+navtitle: Federal Common Policy CA Removal from Apple Trust Stores Impact
+title: Federal Common Policy CA Removal from Apple Trust Stores Impact
+pubDate: June 8, 2018
+collection: announcements
+permalink: announcements/applepkichanges/
+description: Upcoming changes regarding Apple's Root Certificate Program could impact your agency. The Federal PKI Policy Authority has elected to remove our U.S. Government Root CA certificate (Federal Common Policy CA) from the Apple Operating System Trust Stores.  This change will impact government users of Apple iOS, macOS, and tvOS, starting in **September-October 2018**. <br><br> This change will cause government users to receive errors when encountering instances of a Federal PKI CA-issued certificate. You can mitigate the impact for government intranets and government-furnished equipment by using configuration management tools for federal devices.
```

Comment:
tvOS is accurate - but I can't think of a single use case for govt where this would be in use?  :+1: